src/github.js

```diff
@@ -25,7 +26,7 @@ function getEvents(params) {
     let page = 1;
 
     // adjust last created at based on the user provided hours;
-    lastCreatedAt.setHours(lastCreatedAt.getHours() - params.hours);
+    lastCreatedAt.setHours(lastCreatedAt.getHours() - params.hours); // ? 
```

Comment:
I'm updating the `lastCreatedAt` date variable based on the number of hours the user want to see.

If is 1 week then we will do `time.now() - 168` hours.